conf/messages.en

```diff
@@ -246,7 +246,7 @@ secondContactEmail.change.hidden = second contact email address
 registrationConfirmation.title = Registration successful
 registrationConfirmation.heading = Registration successful
 registrationConfirmation.text = Your CBC ID
-registrationConfirmation.p1 = You will need to provide your CBC ID in every file you send to HMRC.
+registrationConfirmation.p1 = You will need to provide your country-by-country (CBC) ID in every file you send to HMRC. You may make a note of your CBC ID, however, it will be in confirmation email we will send you
```

Comment:
I think you've missed "the" and missed the full stop at the end. it should end:

"it will be in **the** confirmation email we will send you."